app/graphql/concerns/validate_submission_gradable.rb

```diff
@@ -3,7 +3,15 @@ module ValidateSubmissionGradable
 
   class OwnersShouldBeActive < GraphQL::Schema::Validator
     def validate(_object, _context, value)
-      submission = TimelineEvent.find(value[:submission_id])
+      submission = TimelineEvent.find_by(id: value[:submission_id])
+
+      if submission.blank?
+        raise GraphQL::ExecutionError,
+              I18n.t(
+                "mutations.create_grading.submission_missing_error",
+                submission_id: value[:submission_id]
+              )
+      end
```

Comment:
>In fact, SubmissionShouldBeLive is doing something wrong - it's first fetching a submission using find_by, but proceeds to call submission.archived? as though find_by will always return a Submission instance.
>
I kept it that way, as validators are executed sequentially then it's guaranteed that `find_by` will always return a submission, in `SubmissionShouldBeLive`,  because the first validator checks the existence of the submission and raises an exception when the submission is not present.